flagd-proxy/cmd/start.go

```diff
@@ -64,10 +68,21 @@ var startCmd = &cobra.Command{
 
 		syncStore := syncStore.NewSyncStore(ctx, logger)
 		s := syncServer.NewServer(logger, syncStore)
+
+		// If --management-port is set use that value. If not and
+		// --metrics-port is set use that value. Otherwise use the default
+		// value.
+		managementPort := uint16(defaultManagementPort)
+		if viper.GetUint16(managementPortFlagName) != defaultManagementPort {
+			managementPort = viper.GetUint16(managementPortFlagName)
+		} else if viper.GetUint16(metricsPortFlagName) != defaultManagementPort {
+			managementPort = viper.GetUint16(metricsPortFlagName)
```

Comment:
Is this actually breaking with the change [here](https://github.com/open-feature/flagd/pull/1012/files#diff-b91a595f5d0f13cb46c493699f061cc8242e7cdef20cb73de82a24fe9b7b3b19R72-R79)?

Reply:
Good idea. Deprecation warning log added. Thanks!

Reply:
@craigpastro Minor change - we can avoid the if clause of the warning log by moving it to this else clause.